components/locid/src/subtags/variants.rs

```diff
@@ -127,7 +123,7 @@ impl Variants {
     ///
     /// This function accepts any [`Box`]`<`[`Variant`]`>` that is expected to be a
     /// valid list of sorted variants.
-    pub const unsafe fn from_raw_unchecked(input: Option<Box<[Variant]>>) -> Self {
+    pub const unsafe fn from_raw_unchecked(input: ShortVec<Variant>) -> Self {
```

Comment:
@zbraniecki These `raw` methods are unused in ICU4X outside tests. Do you have an objection to removing them? Doing so would bring the following benefits:

1. We could keep `ShortVec` private more easily
2. It's weird to have unchecked methods that are public and safe

If you want to keep them, do you have a suggestion on what signature to use such that we can avoid making `ShortVec` be public?

Reply:
@yzhang1994 Please remove the two methods that use ShortVec in their public signature and then move ShortVec back to being crate-private.

Reply:
Done. Remove both methods & made ShortVec crate-private.